Bermuda/Bermuda/MenuGraveyardScreen.cpp

```diff
@@ -0,0 +1,217 @@
+#include "MenuGraveyardScreen.h"
+#include "MenuState.h"
+#include <ctime> // moet verplaatst worden naar player gameover
+
+MenuGraveyardScreen::MenuGraveyardScreen()
+{
+	init();
+}
+
+void MenuGraveyardScreen::init()
+{
+	setBackground();
+	graveyard = new Graveyard;
+	createTable();
+}
+
+void MenuGraveyardScreen::createTable()
+{
+	TTF_Font* font = TTF_OpenFont((RESOURCEPATH + "fonts\\segoeuib.ttf").c_str(), 24);
+	SDL_Color white = { 255, 255, 255 };
+	std::string nameText = "Character name";
+	std::string daysSurvivedText = "Days survived";
+	std::string dayDiedText = "Date of dead";
```

Comment:
"Date of death"

En character names doen we niet aan. Die zou ik dus weglaten.